2021/OffSeason2021/src/main/include/subsystems/Intake.h

```diff
@@ -1,47 +1,74 @@
-// RobotBuilder Version: 3.1
-//
-// This file was generated by RobotBuilder. It contains sections of
-// code that are automatically generated and assigned by robotbuilder.
-// These sections will be updated in the future when you export to
-// C++ from RobotBuilder. Do not put any code or make any change in
-// the blocks indicating autogenerated code or it will be lost on an
-// update. Deleting the comments indicating the section will prevent
-// it from being updated in the future.
-
-#pragma once
-
-// BEGIN AUTOGENERATED CODE, SOURCE=ROBOTBUILDER ID=INCLUDES
-#include "ctre/Phoenix.h"
-
-#include <frc/Solenoid.h>
-#include <frc2/command/SubsystemBase.h>
-
-// END AUTOGENERATED CODE, SOURCE=ROBOTBUILDER ID=INCLUDES
-
-/**
- *
- *
- * @author ExampleAuthor
- */
-class Intake : public frc2::SubsystemBase
-{
-private:
-    // It's desirable that everything possible is private except
-    // for methods that implement subsystem capabilities
-    // BEGIN AUTOGENERATED CODE, SOURCE=ROBOTBUILDER ID=DECLARATIONS
-    frc::Solenoid m_position{ 0, 0 };
-    WPI_TalonSRX m_motorIN6{ 6 };
-
-    // END AUTOGENERATED CODE, SOURCE=ROBOTBUILDER ID=DECLARATIONS
-public:
-    Intake();
-
-    void Periodic() override;
-    void SimulationPeriodic() override;
-    // BEGIN AUTOGENERATED CODE, SOURCE=ROBOTBUILDER ID=CMDPIDGETTERS
-
-    // END AUTOGENERATED CODE, SOURCE=ROBOTBUILDER ID=CMDPIDGETTERS
-    // BEGIN AUTOGENERATED CODE, SOURCE=ROBOTBUILDER ID=CONSTANTS
-
-    // END AUTOGENERATED CODE, SOURCE=ROBOTBUILDER ID=CONSTANTS
-};
+// RobotBuilder Version: 3.1
+//
+// This file was generated by RobotBuilder. It contains sections of
+// code that are automatically generated and assigned by robotbuilder.
+// These sections will be updated in the future when you export to
+// C++ from RobotBuilder. Do not put any code or make any change in
+// the blocks indicating autogenerated code or it will be lost on an
+// update. Deleting the comments indicating the section will prevent
+// it from being updated in the future.
+
+#pragma once
```

Comment:
I think instead of the double solenoid, it's supposed to be a single solenoid, so the code it was before was correct since it was autogenerated from robot builder. I don't think we're planning to use double solenoids in the future

Reply:
Correct that it should be a SingleSolenoid. So this should be changed back. And the health check calls to "IsFwdSolenoidBlackListed" should be changed to "IsBlackListed" since a single solenoid does not have both a forward/reverse position.